### DISCUSSION TITLE
GitHub Student Pack renewal issue

### DISCUSSION THREAD

**Discussion by kausthub-kannan** [community/community#118722] [GitHub Education]
### Hi everyone

I applied for the GitHub Student Developer Pack in March 2023 and was successfully approved. I'm currently still enrolled as a student and would like to renew my access.  **The website mentions a re-verification process (details in the attached screenshot), but it's been nearly 20 days**. Could you advise on the typical wait time for re-verification?

<a href="https://ibb.co/xMndKBh"><img src="https://i.ibb.co/ZhQ7RFd/Screenshot-20240411-232924.png" alt="Screenshot-20240411-232924" border="0"></a>

**Thank you**

**Comment by samus-aran** [comment]
Hi @kausthub-kannan 

You don't need to re-verify unless asked. The system will open up the re-verification application when it's needed. So currently there is nothing you need to do as long as you still have your benefits 👍
